### PR TITLE
Implement end time after start time

### PR DESCRIPTION
**Pull request by joycelynng**
Have not written included this in the testing.
Tests that would be affected are AddCommandTest and EditCommandTest.

### CODE REVIEW

**Comment by kabir-plod** [comment]
End time after start time reviewed

File: src/test/java/guitests/SetStorageCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    public void setStorage() {
+
+        //verify a non-empty list can be cleared
+        assertTrue(taskListPanel.isListMatching(td.getTypicalTasks()));
+        assertSetStorageCommandSuccess();
+
+        //verify other commands can work after a clear command
+        commandBox.runCommand(td.hoon.getAddCommand());
+        assertTrue(taskListPanel.isListMatching(td.hoon));
+        commandBox.runCommand("delete 1");
```
Comment (kabir-plod) [inline src/test/java/guitests/SetStorageCommandTest.java]: Command should be set-storage?

Comment (joycelynng) [inline src/test/java/guitests/SetStorageCommandTest.java]: I think I have accidentally left this uncompleted SetStorageCommandTest in this branch. Will remove and push again.

**Comment by kabir-plod** [comment]
Accidentally brought over a lot of commits... T.T
Fixed the parser, added one test case. Also, handbook says not to add author tags inside a method.

**Comment by kabir-plod** [comment]
Ok @joycelynng I seem to have messed up. I suggest deleting this branch and adding your changes again in a separate one. If you'd like, I'll can do this (since I messed up)

**Pull request closed without merging**